config/enterprise_versions.yml

```diff
@@ -27,18 +27,18 @@ components:
     image: tigera/es-proxy
     version: v3.0.0-0.dev-29-gfe7b046
   eck-kibana:
-    version: 7.3.2
+    version: 7.6.2
   kibana:
-    image: tigera/kibana
-    version: v3.0.0-0.dev-0-gddc3967
+    image: kibana/kibana
```

Comment:
Does this need to specify registry: as well?

Reply:
I think registry is picked up from https://github.com/tigera/operator/pull/582/files#diff-48f4115f670d2f499a32f8391d776314R40